app/models/bgeigie_import.rb

```diff
@@ -364,4 +365,30 @@ def nmea_to_lat_lng(latitude_nmea, _north_south_indicator, longitude_nmea, _east
   def maximum_cpm
     bgeigie_logs.maximum(:cpm)
   end
+
+  def minimum_cpm
+    # used to check if the drive contains cpm value lower than or equal to 0
+    bgeigie_logs.minimum(:cpm)
+  end
+
+  def check_auto_approve
+    # run each auto approval rule and
+    # update would_auto_approve column based on if all rules passed
+    ap_no_zero_cpm
```

Comment:
You might want to use `update_attributes` instead of `update_column` because (1) `update_attributes` updates `updated_at` column and (2) can update multiple columns.

```ruby
update_attributes(
  would_auto_approve: auto_apprv_no_zero_cpm,
  auto_apprv_no_zero_cpm: minimum_cpm.positive?
)
```

Reply:
I had some trouble using the update_attributes method so I use two update_attribute instead